rpcs3/Emu/Io/usio.cpp

```diff
@@ -214,6 +276,15 @@ void usb_device_usio::translate_input()
 	translate_from_pad(0, 0);
 	translate_from_pad(1, 1);
 
+	// Like some of namcos other arcade games, taikos test switch is a physical switch and not a button
+	if (is_test)
+	{
+		std::memcpy(input_buf.data() + 0, &button_test, sizeof(u8));
```

Comment:
Separating the handling of Test Switch and the other switches is not necessary.

Reply:
Your suggestion looks a lot nicer so it's probably better to just merge your PR.